packages/contracts-bedrock/.gas-snapshot

```diff
@@ -291,9 +291,6 @@ L2StandardBridge_Test:test_initialize_succeeds() (gas: 26316)
 L2StandardBridge_Test:test_receive_succeeds() (gas: 177125)
 L2StandardBridge_Test:test_withdraw_ether_succeeds() (gas: 143366)
 L2StandardBridge_Test:test_withdraw_insufficientValue_reverts() (gas: 16552)
-L2ToL1MessagePasserTest:test_burn_succeeds() (gas: 112530)
-L2ToL1MessagePasserTest:test_initiateWithdrawal_fromContract_succeeds() (gas: 70343)
-L2ToL1MessagePasserTest:test_initiateWithdrawal_fromEOA_succeeds() (gas: 75830)
```

Comment:
Hm wonder if there's a way to somehow provide reasonable fuzz bounds in a gas snapshot output